src/cli.rs

```diff
@@ -163,11 +167,14 @@ pub fn run() -> Result<()> {
 
     match &opts.path {
         None => {
-            if !opts.rekey {
-                bail!("agenix requires a path argument (unless rekeying).")
+            if !opts.rekey && !opts.validate_config {
+                bail!("agenix requires a path argument (unless rekeying or validating the config).")
             }
         }
         Some(path) => {
+            if opts.validate_config {
```

Comment:
I don't think this is necessary -- we just won't use the provided path. Giving a path won't affect anything. Maybe a warning instead, if you still feel inclined to notify the user.